docs/docsite/rst/conf.py

```diff
@@ -238,15 +238,15 @@
     'current_version': version,
     'latest_version': (
         'devel' if tags.has('all') else
-        '2.16' if tags.has('core_lang') or tags.has('core') else
-        '9' if tags.has('ansible')
+        '2.17' if tags.has('core_lang') or tags.has('core') else
+        '10' if tags.has('ansible')
         else '<UNKNOWN>'
     ),
     # list specifically out of order to make latest work
     'available_versions': (
         ('devel',) if tags.has('all') else
         ('2.15_ja', '2.14_ja', '2.13_ja',) if tags.has('core_lang') else
```

Comment:
@samccann I guess we need to leave this line unchanged as 2.15 is the last version that includes a Japanese translation. Does that make sense?

Reply:
So two things going on here:
1 - I forgot to EOL the japanese 2.13, so this could go in a separate PR and be merged, then backported all the way down to 2.13... then we republish the japanese docs 2-13 through 2.15.

2. We then repeat this effort after 2.17 releases, to remove 2.14 similarly from the japanese docs.

My nickel would be to just bunch them both together in a separate PR and don't merge it til release day (and then backport all the way down to 2.13 and republish.

We'd also have to EOL 2.13 and 2.14 japanese docs... That's a separate PR that only goes into those individual stable branches.

Reply:
I've created a separate issue for this so we don't forget it: https://github.com/ansible/ansible-documentation/issues/1255